src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -1003,10 +1003,10 @@ ul.packages .metadata-block:last-child {
 .package-aside .details {
   padding-left: 0;
 }
-.package .package-aside i, .package .readme-expander i {
+.package .package-aside i:not(.fa), .package .readme-expander i:not(.fa) {
```

Comment:
I'm not using Glyphicon but FontAwesome that provide much icons to feed my needs: http://fontawesome.io/icons/

BTW, using `.glyphicon` on the view with produce the same gray color issue.